plutus-core/plutus-core/src/Data/ByteString/Hash.hs

```diff
@@ -1,9 +1,9 @@
 -- | Hash functions for lazy [[Data.ByteString.ByteString]]s
 {-# LANGUAGE TypeApplications #-}
 module Data.ByteString.Hash
-    ( sha2
-    , sha3
-    , blake2b
+    ( sha2_256
```

Comment:
I thought I might as well change these for consistency while I was at it (although [Cardano.Crypto.Hash](https://github.com/input-output-hk/cardano-base/tree/master/cardano-crypto-class/src/Cardano/Crypto/Hash) (where the underlying implementations are) is also inconsistent about the names of the SHA functions (`SHA256` and `SHA3_256`)),